coding-concepts.js

```diff
@@ -69,5 +80,7 @@ class Learn {
 const learnStudent = new Learn("George")
 // console.log(learnStudent)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: 'George', 'Echo', 2022
+// b) Verify and explain: 
+      // After verifying, the output is an object => Learn { student: 'George', cohort: 'Echo', year: 2022 }.
```

Comment:
yes new is allowing you to instantiate the class, create a new instance of the Learn class, which is your object learnStudent